static/sass/styles.scss

```diff
@@ -8,6 +8,7 @@ $color-navigation-background: #252525;
 
 @include vf-base;
 
+@include vf-u-hide; // required by vf-p-icons
```

Comment:
Needed because of https://github.com/vanilla-framework/vanilla-framework/issues/1821

Reply:
@bartaz Hotfixed as part of 1.7.1 release